app/src/main/java/org/oppia/app/recyclerview/BindableAdapter.kt

```diff
@@ -41,13 +42,16 @@ class BindableAdapter<T : Any> internal constructor(
   private fun setData(newDataList: List<T>) {
     val result = DiffUtil.calculateDiff(
       RecyclerDataDiffCallback(dataList, newDataList),
-      false
+      /* detectMoves*/ false
     )
     dataList.clear()
     dataList += newDataList.toMutableList()
     result.dispatchUpdatesTo(this)
   }
 
+  @VisibleForTesting(otherwise = VisibleForTesting.NONE)
```

Comment:
Prefer not exposing a public API for the purposes of testing. Could we instead bind a test recycler view using the adapter & verify that the expected elements are bound by using Espresso matchers?

Reply:
Sure

Reply:
Resolving this now on behalf of @BenHenning